src/eslint-config-adeira/src/presets/react.js

```diff
@@ -211,6 +212,40 @@ module.exports = ({
     'jsx-a11y/scope': ERROR,
     'jsx-a11y/tabindex-no-positive': ERROR,
   },
+  overrides: [
+    {
+      // React Testing Library (https://github.com/testing-library/eslint-plugin-testing-library)
+      files: ['**/__tests__/**/*.[jt]s?(x)', '**/?(*.)+(spec|test).[jt]s?(x)'],
+      rules: {
+        'testing-library/await-async-query': OFF, // TODO: seems to be broken for some `findAllByProps` in our codebase
+        'testing-library/await-async-utils': NEXT_VERSION_ERROR,
+        'testing-library/await-fire-event': OFF, // Vue only (https://github.com/testing-library/eslint-plugin-testing-library#supported-rules)
+        'testing-library/consistent-data-testid': OFF,
+        'testing-library/no-await-sync-events': OFF,
+        'testing-library/no-await-sync-query': NEXT_VERSION_ERROR,
+        'testing-library/no-container': OFF,
+        'testing-library/no-debug': NEXT_VERSION_ERROR,
+        'testing-library/no-dom-import': [NEXT_VERSION_ERROR, 'react'],
+        'testing-library/no-manual-cleanup': OFF,
+        'testing-library/no-node-access': OFF,
+        'testing-library/no-promise-in-fire-event': NEXT_VERSION_ERROR,
+        'testing-library/no-render-in-setup': OFF,
+        'testing-library/no-unnecessary-act': OFF,
```

Comment:
Wouldn't this be of help?

Reply:
It would. I changed it to `NEXT_VERSION_ERROR`. The thing is that there are many other rules from `testing-library` that should be enabled but my thinking is to do it gradually to ease the migration. I am also planning to include https://github.com/testing-library/eslint-plugin-jest-dom, but again: better to do it gradually in the next release for example. Hopefully, that makes sense.